packages/ERTP/src/types-ambient.js

```diff
@@ -211,6 +220,33 @@
  *   Payment containing newly minted amount.
  */
 
+// /////////////////////////// Purse / Payment /////////////////////////////////
+
+/**
+ * Issuers first became durable with recovery sets and no option to suppress
+ * them. Thus, absence of a `RecoverySetsOption` state is equivalent to
+ * `'hasRecoverySets'`. By contrast, the absence of `RecoverySetsOption` provide
+ * parameter defaults to the predecessor's `RecoverySetsOption` state, or
+ * `'hasRecoverySets'` if none.
```

Comment:
This difference is more clearly represented by the distinction between variables labeled `recoverySetsState` and `recoverySetsOption`. If there is no previously stored recoverySetsState, then the implicit recoverySetsState is considered equivalent to `'hasRecoverySets'` (and made explicit in `oldRecoverySetsState`). If there is no provided `recoverySetsOption` in the optional `options` parameter typed `IssuerOptionsRecord`, especially in the optional `options` parameter of `reallyProvideIssuerKit`, then this absence defaults to preserving `oldRecoverySetsState` as the new `recoverySetsState`. Otherwise, the new `recoverySetsState` is made the same as the `recoverySetsOption` if possible (and errors in not possible).

Reply:
The logic implementing this defaulting logic is in `upgradeIssuerKit`, but it should be understood at all call sites upstream of it with a `IssuerOptionsRecord`-typed `options` parameter or simply an optional `recoverySetsOption` parameter.

Reply:
Yes, fwiw, "defaults" is the verb of that bad sentence :( .